types/jsonwebtoken-promisified/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
I need some justification for why you've disabled all the lint rules. We have them all in place for a reason.

Reply:
I see what you mean. I copied the tsconfig from types/jsonwebtoken.

Reply:
Thanks for catching that. The linting caught a lot of stupid stuff.  I'm fixing it.

I looked at some of the nearby folders, and noticed ~50% of them have the same tsconfig. I suspect DefinitelyTyped is littered with this bad pattern. 

E.g,: jsonnet, jsonpath jsonstream, jspdf, jsrender

Reply:
Ah, I see what's happening. The javascript has function signatures that aren't easily matched by typescript rules. I believe the underlying library must be doing runtime inspection of arguments. So enabling the linting rules creates a bunch of errors.